src/translations/en.json

```
@@ -64,16 +64,8 @@
           "rank": "By Rank"
         }
       },
-      "cannotCounterDialog": {
-        "opponentPlayed": "Your opponent has played the ",
-        "oneOff": "as a one-off",
-        "target": "targeting ",
-        "counter": "to Counter",
-        "cannotCounter": "You cannot Counter, because ",
-        "reasons": {
-          "opponentWithQueen": "your opponent has a queen",
-          "noTwoPresent": "you do not have a two"
-        }
+      "threeDialog": {
+        "title": "Select a Card from Scrap",
```

```suggestion
        "title": "Select a Card from Scrap"
```
I think this is the issue. The trailing comma on the last line of a block is invalid json